### DISCUSSION TITLE
Help with GPS on raspberry Pi and PureMaps

### DISCUSSION THREAD

**Discussion by karlo922** [rinigus/pure-maps#670] [Q&A]
Hi,
I've got gpsd and cgps running on my RPi4 with a serial attached GPS receiver. 
The output of cgps is fine, but no position is shown within PureMaps.

What do I need to do to get PureMaps using my GPS device?

**Comment by rinigus** [comment]
You need to have Qt Location plugin support for this GPS source. As soon as you configure it to work with Qt, Pure Maps should pick it up as well.
